worker/scraper.py

```diff
@@ -37,14 +38,14 @@ def scrape_and_categorize_articles():
         # Extract text and categorize item
         text_only = soup.get_text()
         categories = categorizer.categorize_item(item.title, text_only)
-        if len(categories) >= 1:
-            item.category_1 = categories[0]
+        for category in categories:
+            item.categories.append(category)
 
-        if len(categories) >= 2:
-            item.category_2 = categories[1]
+        item.categorized = True
 
         db.session.commit()
-        logger.info("CATEGORIZED [%s]: (%s, %s)" % (item.title, item.category_1, item.category_2))
+
+        logger.info("CATEGORIZED [%s]" % (item.title))
```

Comment:
Can you print the categories too? Perhaps do a ```map```/```.join``` on the list of category strings and then add them to the logger? I think it's useful information to render to the log and is worth the bit of computational effort.